api/agent/drivers/docker/docker.go

```diff
@@ -102,11 +125,27 @@ func NewDocker(conf drivers.Config) *DockerDriver {
 		}
 	}
 
+	imageEvictChan := make(chan docker.APIImages)
+	onEvictor := OnImageEvict(imageEvictChan)
+	go NewImageCleaner(driver, imageEvictChan)
+
+	//20GB
+	driver.imageLRU = NewLRU(1024*1024*1024*1024*20, onEvictor)
+
 	if conf.DockerLoadFile != "" {
 		err = loadDockerImages(driver, conf.DockerLoadFile)
 		if err != nil {
 			logrus.WithError(err).Fatalf("cannot load docker images in %s", conf.DockerLoadFile)
 		}
+
+		ctx, _ := common.LoggerWithFields(context.Background(), logrus.Fields{"stack": "listDockerImages"})
+		images, err := driver.docker.ListImages(ctx)
```

Comment:
Oh, loadDockerImages() should be excluded from LRU evictor.

Reply:
Ahh, because we never want to delete any that we load?

Reply:
Yes, these are special, we should not touch them. :-)